src/core/components/highlight-code.jsx

```diff
@@ -39,6 +39,18 @@ export default class HighlightCode extends Component {
     }
   }
 
+  componentDidMount() {
+    [this.syntaxHighlighter, this.pre]
+    .filter(element => !!element)
+    .map(element => element.addEventListener("mousewheel", this.preventYScrollingBeyondElement, { passive: false }))
+  }
+
+  componentWillUnmount() {
+    [this.syntaxHighlighter, this.pre]
+    .filter(element => !!element)
+    .map(element => element.removeEventListener("mousewheel", this.preventYScrollingBeyondElement))
```

Comment:
```suggestion
    .map(element => element?.removeEventListener("mousewheel", this.preventYScrollingBeyondElement))
```
Could be shortened one more time.

Reply:
@mathis-m thanks for additional suggestions.